views/random-beer.hbs

```diff
@@ -0,0 +1,28 @@
+<div><h1>Your Random Beer here! Cheers!</h1></div>
+
+{{#RandomB}}
```

Comment:
This works but you can also use #each since you are passing an array (although an array of just one element). You can skip this and just use `RandomB` if you just pass the element instead of the array from app.js
Example:
`{{>beerpartial RandomB}}`
`{{#each RandomB.food_pairing}}`

If you are passing the element and want to set the scope once you could use `{{#with RandomB}}` and then you can do: 
`{{>beerpartial this}}`
`{{#each food_pairing}}`